Source/MSBuildTools.Unity/Packages/com.microsoft.msbuildforunity/Editor/ProjectGenerator/MSBuildTemplates/Plugin.iOS.meta.template

```diff
@@ -1,11 +1,17 @@
 fileFormatVersion: 2
-#PROJECT_GUID_TEMPLATE guid: <PROJECT_GUID_TOKEN>
+#PROJECT_GUID_TEMPLATE_START
+guid: <PROJECT_GUID_TOKEN>
+#PROJECT_GUID_TEMPLATE_END
 PluginImporter:
   externalObjects: {}
   serializedVersion: 2
   iconMap: {}
-#EXECUTION_ORDER_TEMPLATE   executionOrder: <EMPTY_TOKEN>
-#EXECUTION_ORDER_ENTRY_TEMPLATE     <SCRIPT_FULL_NAME_TOKEN>: <SCRIPT_EXECUTION_VALUE_TOKEN>
+#EXECUTION_ORDER_TEMPLATE_START
+  executionOrder: <EMPTY_TOKEN>
+#EXECUTION_ORDER_TEMPLATE_END
+#EXECUTION_ORDER_ENTRY_TEMPLATE_START
+    <SCRIPT_FULL_NAME_TOKEN>: <SCRIPT_EXECUTION_VALUE_TOKEN>
```

Comment:
Not sure if this is intentional or not, but this line has four spaces while the ones above (`iconMap`, `executionOrder`, etc) all have two.

Reply:
That is intentional, it's a list from above :D